test/shared/src/test/scala/zio/test/BaseSpec.scala

```diff
@@ -0,0 +1,5 @@
+package zio.test
+
+trait BaseSpec {
```

Comment:
Can we have `BaseSpec` extend `DefaultRuntime` so Specs just have to extend one thing?

Reply:
Sure, but three Specs don't extend `DefaultRuntime` (`AssertionSpec`, `GenSpec` and `GenZIOSpec`). If it's okay that they inherit the `DefaultRuntime`, than sure I can let `BaseSpec` extends `DefaultRuntime`.

An other option would be to have a third trait that extends both `DefaultRuntime` and `BaseSpec`, but in tests I rather prefer a bit of duplication.

Reply:
Yes I think it is fine to have them all extent `DefaultRuntime`. It is a little functionality they won't need but I think that is outweighed by the benefit of just having one thing contributors need to extend when adding new Specs. Could we also rename it `ZIOBaseSpec`? That should trigger the CI to run again as well.

Reply:
Done